src/actions/archive.js

```diff
@@ -36,14 +36,15 @@ function archiveAction(command, options) {
 
         // Exclude destination file from archive
         const destFile = path.basename(command.destination);
-        const globOptions = Object.assign({ ignore: destFile }, command.options.globOptions || {});
+        const ignore = (command.options && command.options.ignore || []).concat(destFile);
```

Comment:
here we should ignore the `ignore` option` if it is a string. and should add a note of it in the README.

Reply:
there is no instructions about `options.ignore` in the README, can you supplement it?

Reply:
`  {
                  source: './dist',
                  destination: './dist/pc.zip',
                  options: {
                    ignore: ['lib/*', 'lib/']
                  }
                }
              ]`
worked

Reply:
I was just fixing it. I will publish a new version shortly. Ignore should be actually inside `globOptions`.

Reply:
Fixed in v6.1.3. The options should work as expected now.

Reply:
how should I use it? my test just work on first directory, if there is other directories, that's still not ignored.

Reply:
well,`ignore: ['lib/*', 'lib/*/**', 'lib/']` seems work

Reply:
that was never meant to be there. Starting v6.1.3. it wont work. It should be like. Usage it same, just that it is moved into globOptions.

```jsonc
{
  "options": {
    // other archiver options
    "globOptions": {
      // https://github.com/Yqnn/node-readdir-glob#options
      "dot": true,
      "ignore": ["lib/*", "lib/*/**", "lib/"]
    }
  }
}
```